src/_locales/ru/messages.json

```diff
@@ -304,5 +304,16 @@
 	"browse_data_search": { "message": "Поиск в собранных данных" },
 	"content_script_width": { "message": "150px !important" },
 	"parent_selector_self_citation": { "message": "Селектор не может указывать только на себя" },
-	"sitemap_export_copy_text_button": { "message": "Скопировать карту обхода в буфер обмена" }
+	"sitemap_export_copy_text_button": { "message": "Скопировать карту обхода в буфер обмена" },
+	"modal_confirm_action_submit": { "message": "Подтвердить" },
+	"modal_confirm_action_cancel": { "message": "Отменить" },
+	"modal_confirm_action_title_delete_selector": {
+		"message": "Вы действительно хотите удалить селектор  <span id='selector-id'></span>?"
+	},
+	"modal_confirm_action_child_delete": {
+		"message": "У удаляемого вами селектора имеются дочерние селекторы <span id='child-count'></span> шт, которые будут удалены"
+	},
+	"modal_confirm_action_title_delete_sitemap": {
+		"message": "Вы действительно хотите удалить карту обхода <span id='sitemap-id'></span>?"
+	}
```

Comment:
Several suggestions
- Use parentheses with number of selectors: `(<span id='child-count'></span> шт.)` in ru/en
- For cancel button we could use `Отмена` instead of `Отменить` since everywhere else its `Отмена`
- For submit button in en I'd use `Confirm`, or `Delete`/`Удалить` in en/ru
- I'd shorten message about children `У удаляемого селектора имеются дочерние (n шт.), ...` so it fits in one line